mllib/src/main/scala/org/apache/spark/ml/classification/DecisionTreeClassifier.scala

```diff
@@ -85,18 +82,16 @@ final class DecisionTreeClassifier
   }
 
   /** (private[ml]) Create a Strategy instance to use with the old API. */
```

Comment:
Yeah, I put it there since the Java docs don't handle package private correctly.  Should I remove it?